src/viur/core/session.py

```diff
@@ -207,6 +207,13 @@ def items(self) -> 'dict_items':
         """
         return self.session.items()
 
+    def setdefault(self, key: str, value: t.Any) -> t.Any:
+        """
+            Set a default value in the current session
+        """
+        self.session.setdefault(key, value)
+        return value
```

Comment:
Cloud be more simple, and this returns the correct value if this key was already in the session.
```suggestion
        return self.session.setdefault(key, value)
```